documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -245,6 +257,17 @@ Note that the Domain custom resource definition is shared.
 Do not delete the CRD if there are other operators in the same cluster
 or you have running domain resources.
 
+Beginning with Operator version 4.0, uninstalling an operator also removes the conversion webhook
+ deployment and associated resources in the same namespace.
+Therefore, if you have multiple operators running in the Kubernetes cluster and want to prevent the removal of the 
```

Comment:
Therefore, if you have multiple operators running, then, by default, an uninstall of one operator will affect the other operators. In addition, the uninstall will disable access to the domain CRD so you will be unable to create new domains. If you want to prevent the uninstall of an operator from having these side effects, then use one of the following options:

Reply:
Changed. Note that `V9` domain resources continue to work even if the conversion webhook is not running or conversion definition in domain CRD is incorrect. The issue happens only when creating domain with `V8` schema.